rust/src/lightning.rs

```diff
@@ -183,8 +183,7 @@ pub async fn open_channel(
             .map_err(|e| anyhow!("Could not create channel with {peer_info} due to {e:?}"))?,
     };
 
-    tracing::debug!("Created channel with {peer_info}");
-    tracing::info!("Channel has been successfully created");
+    tracing::info!("Created channel with {peer_info}");
```

Comment:
🔧 Not a problem with your PR, but the message has always been misleading.

```suggestion
    tracing::info!("Started channel creation with {peer_info}");
```

Reply:
fixed :)